rllib/BUILD

```diff
@@ -1858,6 +1858,27 @@ py_test(
     srcs = ["core/rl_module/tests/test_rl_module_specs.py"]
 )
 
+py_test(
+    name = "test_tf_mlp_head",
+    tags = ["team:rllib", "core", "models"],
+    size = "small",
+    srcs = ["core/models/tf/tests/test_mlp_head.py"]
```

Comment:
@kouroshHakha Added this since your last review. I'll merge these tests once we have feature parity but will only create torch tests for the next models for now and leave this as is.